src/components/StructuredNavigation/StructuredNavigation.js

```diff
@@ -131,13 +131,6 @@ const StructuredNavigation = () => {
         playerDispatch({
           type: 'resetClick',
         });
-        // Ensure autoplay from inaccessible items in playlists
```

Comment:
This is undoing https://github.com/samvera-labs/ramp/pull/569.  Did you test that case (clicking on a playlist item in a playlist disables autoplay without untoggling the switch https://github.com/samvera-labs/ramp/issues/551) is resolved without this code and your changes in VideoJSPlayer.js?

Reply:
The changes in #569 would always play the clicked (selected) item after an an inaccessible item regardless, whether the user had initiated playback at any point, which I think is undesirable behavior. Because it is similar to auto-play without any indication to the user, which violates accessibility guidelines.

And testing this with changes in this PR;
- plays the clicked item after an inaccessible item if the user has initiated playback before
- do not play the clicked item after an inaccessible item if the user has not initiated playback before reaching the inaccessible item

Does this still sound like what we need or does this behavior need to change? 

Because from the description in #551 I can't figure out whether we need to play items automatically after an inaccessible item count down ends without user initiating playback before they click on an inaccessible item.

Reply:
I tested this out and as far as I can tell it is working as I would expect and I'm not able to reproduce #580 or #551.